### PR TITLE
Godstime calculator UI challenge 

### PR DESCRIPTION
**Pull request by Godstyme**
My own design challenge
Here is the link to the live demo https://godstyme.github.io/calculator/

### CODE REVIEW

**Comment by Chiagozielam** [comment]
Your PR name should be descriptive of the PR

**Comment by ebenezerdon** [comment]
HI @Godstyme, can you add a link to your hosted app in the PR description?

**Comment by Godstyme** [comment]
Hi @ebenezerdon, check it out.
I have added  the live demo link.

File: assets/css/app.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    justify-content: center;
+    padding: 10px 0px;
+}
+button.btn-operator{
+    background: #005552;
+}
+button.btn-number{
+    background: #202020;
+}
+
```
Comment (ebenezerdon) [inline assets/css/app.css:51]: Just one empty line is enough.

File: assets/css/app.css [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+button{
+    color: #fff;
+    padding: 30px 0px;
+    text-align: center;
+    cursor: pointer;
+    border-radius: 5px;
+    outline: none;
+    border: none;
+}
+main > section > div.btn-holder > div.btn-row{
```
Comment (ebenezerdon) [inline assets/css/app.css:38]: Why not style the class directly?

File: assets/css/app.css [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    width: 40%;
+}
+main > section > div.screen{
+    padding: 90px 50px 30px 50px;
+    background: #202020;
+    font-weight: bold;
+    text-align: right;
+    font-size: 1.4em;
+    color: #fff;
+}
```
Comment (ebenezerdon) [inline assets/css/app.css:23]: You'll need to add an empty line after each block.

**Review by ebenezerdon — CHANGES_REQUESTED** [review]
Hi @Godstyme, nice attempt. See if you can work on the requested changes.

**Comment by ebenezerdon** [comment]
Hi Godstime, you've made a really good attempt. So far, your buttons are not clickable. See if you can fix this.

**Pull request closed without merging**